rustls/tests/api.rs

```diff
@@ -3412,6 +3412,118 @@ mod test_quic {
     }
 } // mod test_quic
 
+#[test]
```

Comment:
It would be good, for these tests that test conformance with the spec, to also cite what part of the spec compliance is being tested for.